test/v1/connectors/CreateAndAmendOtherDeductionsConnectorSpec.scala

```diff
@@ -16,9 +16,9 @@
 
 package v1.connectors
 
-import api.models.domain.TaxYear
-import v1.models.domain.Nino
-import v1.models.outcomes.ResponseWrapper
+import api.connectors.ConnectorSpec
```

Comment:
Can we update the `body` on line 70 to be `CreateAndAmendOtherDeductionsBody(None)` as the connector spec doesn't touch the body itself so its good practice :)